sormas-ui/src/main/java/de/symeda/sormas/ui/samples/pathogentestlink/PathogenTestList.java

```diff
@@ -38,20 +39,36 @@ public class PathogenTestList extends PaginationList<PathogenTestDto> {
 	private static final int MAX_DISPLAYED_ENTRIES = 5;
 
 	private final SampleReferenceDto sampleRef;
+	private final EnvironmentSampleReferenceDto environmentSampleRef;
 	private final Consumer<Runnable> actionCallback;
 	private final boolean isEditable;
 
 	public PathogenTestList(SampleReferenceDto sampleRef, Consumer<Runnable> actionCallback, boolean isEditAllowed) {
 		super(MAX_DISPLAYED_ENTRIES);
 
 		this.sampleRef = sampleRef;
+		this.environmentSampleRef = null;
+		this.actionCallback = actionCallback;
+		this.isEditable = isEditAllowed;
+	}
+
+	public PathogenTestList(EnvironmentSampleReferenceDto environmentSampleRef, Consumer<Runnable> actionCallback, boolean isEditAllowed) {
+		super(MAX_DISPLAYED_ENTRIES);
+
+		this.sampleRef = null;
```

Comment:
this.sampleRef is null be default so no need to re-initialize it with null

Reply:
removed